extensions/sql-database-projects/package.json

```diff
@@ -26,7 +26,8 @@
   },
   "extensionDependencies": [
     "Microsoft.mssql",
-    "Microsoft.schema-compare"
+    "Microsoft.schema-compare",
+    "Microsoft.data-workspace"
```

Comment:
I added this so that the data-workspace extension will get activated when a sql projects command is run, like "Create project from database". The package.vscode.json for sql-database-projects already has data-workspace as an extension dependency